server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGates.java

```diff
@@ -87,6 +87,7 @@ public QualityGateDto copy(long sourceId, String destinationName) {
     }
   }
 
+  @Deprecated // GJT
```

Comment:
I find it quite useful to add a Javadoc when deprecating method to know which method must be used instead of the deprecated one with a @link pointing to it.

Reply:
Yep, but I found it even more useful to delete such method as soon as possible :)